test/merge-patient-lab-iips-mocha.js

```diff
@@ -8,181 +9,146 @@ chai.use(chaiAsPromised);
-                        .then(commonTest.createComponent).then(function(component){
-
-                            // initialize state
-                            component.rpf.vni().outputState({
-                                data: { id: '001', 
-                                         name: 'Alice', 
-                                         dob: '1979-01-23', 
-                                         glucose: '75',  
-                                         date: '2012-02-01'
-                                 },
-                                lm: 'LM1328113669.00000000000000001'
-                            });
-
-                            var currentState = component.rpf.vni().outputState();
-                            currentState.should.be.an('object');
-                            currentState.should.have.all.keys( 'data', 'lm' );
-                        });
-                });
+                var currentState = node.vni().outputState();
+                currentState.should.be.an('object');
+                currentState.should.have.all.keys( 'data', 'lm' );
+            });
 
-                it("should have an output state data after input", function() {
-                    return Promise.resolve(componentFactory.getComponent )
-                        .then(commonTest.createComponent).then(function(component){
-
-                            // initialize state
-                            var outputState = {
-                                data: {
-                                    id: '001',
-                                    name: 'Alice',
-                                    dob: '1979-01-23',
-                                    glucose: '75',
-                                    date: '2012-02-01'
-                                },
-                                lm: 'LM1328113669.00000000000000001'
-                            };
-                            component.rpf.vni().outputState( _.mapObject(outputState, _.clone) );
-
-                            var currentState = component.rpf.vni().outputState();
-                            currentState.should.be.an('object');
-                            currentState.data.should.deep.equal( outputState.data );
-                            currentState.lm.should.be.a('string');
-                        });
-                });
+            it("should have an output state data after input", function() {
+                var node = test.createComponent(compFactory);
+                // initialize state
+                var outputState = {
+                    data: {
+                        id: '001',
+                        name: 'Alice',
+                        dob: '1979-01-23',
+                        glucose: '75',
+                        date: '2012-02-01'
+                    },
+                    lm: 'LM1328113669.00000000000000001'
+                };
+                node.vni().outputState( _.mapObject(outputState, _.clone) );
```

Comment:
JS noobie question: could this line just as well have been written ass
<pre>
node.vni().outputState( _.clone(outputState) );
</pre>